### ISSUE TITLE
Adding translated strings for Sinhala (ISO 639-1 code: si)

### ISSUE THREAD

**Issue by Chevindu** [squidfunk/mkdocs-material#2292]
## Instructions

1. Check, if your language is already available: [here](https://bit.ly/33vFDD0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "si",
  "direction": "ltr",
  "clipboard.copy": "කොපි කරන්න",
  "clipboard.copied": "කොපි කළා",
  "edit.link.title": "පිටුව සංස්කරණය",
  "footer.previous": "පසුගිය",
  "footer.next": "මීළඟ",
  "footer.title": "පාදම",
  "header.title": "ශීර්ෂය",
  "meta.comments": "ප්‍රතිචාර",
  "meta.source": "මූලාශ්‍රය",
  "nav.title": "යාත්‍රණය",
  "search.config.lang": "si",
  "search.config.pipeline": "trimmer, stopWordFilter",
  "search.config.separator": "[\s\-]+",
  "search.placeholder": "සොයන්න",
  "search.reset": "මකන්න",
  "search.result.placeholder": "සෙවීමට ටයිප් කරන්න",
  "search.result.none": "කිසිවක් හමු නොවුණි",
  "search.result.one": "1 ගැලපෙන ගොනුවක්",
  "search.result.other": "ගැලපෙන ගොනු # ක්",
  "search.result.more.one": "තව 1 ප්‍රතිඵලයක්",
  "search.result.more.other": "තව ප්‍රතිඵල # ක්",
  "skip.link.title": "අන්තර්ගතය වෙත යන්න",
  "source.link.title": "රිපොසිටරියට යන්න",
  "source.revision.date": "අවසන් යාවත්කාලීන වීම",
  "tabs.title": "ටැබ්ස්",
  "toc.title": "පටුන"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks!

**Comment by squidfunk** [comment]
Added in 5dc7a79e07e325afea7b92d2b77bcd0622277721 and released as part of 7.0.0b2.
